chatgpt_wrapper/chatgpt.py

```diff
@@ -54,7 +60,9 @@ def __init__(self, headless: bool = True, browser="firefox", timeout=60, proxy:
         self._start_browser()
         self.parent_message_id = str(uuid.uuid4())
         self.conversation_id = None
+        self.conversation_title_set = None
```

Comment:
could you please rename this to clarify it's binary:
```suggestion
        self.is_conversation_title_set = False
```